apps/pgsql/patroni/openshift/deployment.yaml

```diff
@@ -220,7 +230,7 @@ objects:
         - name: ${NAME}${SUFFIX}
```

Comment:
I think this can be removed `volumes` as `statefulset.spec.volumeClaimTemplates` are implied `statefulset.spec.template.spec.volumes`.